components/VersionHistory/VersionHistory.vue

```diff
@@ -4,20 +4,23 @@
     :show-close="false"
     class="version-history-dialog"
     width="772px"
-    height="600px"
+    height="360px"
```

Comment:
Should these distances be units of `em` or `rem`?

Reply:
Thanks Hugh, happy to switch all of these to `em`